jupyter_server/event_schemas/contents_service/v1.yaml

```diff
@@ -0,0 +1,80 @@
+"$id": https://events.jupyter.org/jupyter_server/contents_service/v1
+version: 1
+title: Contents Manager activities
+personal-data: true
+description: |
+  Record actions on files via the ContentsManager.
+
+  The notebook ContentsManager REST API is used by all frontends to retreive,
+  save, list, delete and perform other actions on notebooks, directories,
+  and other files through the UI. This is pluggable - the default acts on
+  the file system, but can be replaced with a different ContentsManager
+  implementation - to work on S3, Postgres, other object stores, etc.
+  The events get recorded regardless of the ContentsManager implementation
+  being used.
+
+  Limitations:
+
+  1. This does not record all filesystem access, just the ones that happen
+     explicitly via the notebook server's REST API. Users can (and often do)
+     trivially access the filesystem in many other ways (such as `open()` calls
+     in their code), so this is usually never a complete record.
+  2. As with all events recorded by the notebook server, users most likely
+     have the ability to modify the code of the notebook server. Unless other
+     security measures are in place, these events should be treated as user
+     controlled and not used in high security areas.
+  3. Events are only recorded when an action succeeds.
+type: object
+required:
+  - action
+  - path
+properties:
+  action:
+    enum:
+      - get
+      - create
+      - save
+      - upload
+      - rename
+      - copy
+      - delete
+    description: |
+      Action performed by the ContentsManager API.
+
+      This is a required field.
+
+      Possible values:
+
+      1. get
+         Get contents of a particular file, or list contents of a directory.
+
+      2. create
+         Create a new directory or file at 'path'. Currently, name of the
+         file or directory is auto generated by the ContentsManager implementation.
+
+      3. save
+         Save a file at path with contents from the client
+
+      4. upload
+         Upload a file at given path with contents from the client
```

Comment:
It looks like the upload event is not being emitted anywhere in the ContentsManager. Not sure if this event is necessary either (see above comment on anticipating use-cases).

Reply:
:+1: removed.